src/pages/HomePage/index.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Link } from "react-router-dom";
+import SidebarLeft from "../../components/SidebarLeft";
+import SidebarRight from "../../components/SidebarRight";
+import {
+  setBooksCollections,
+  fetchFullData,
+} from "../../store/homepage/actions";
+import { selectBooksCollections } from "../../store/homepage/selectors";
+import "./index.css";
+
+export default function HomePage() {
+  const dispatch = useDispatch();
+  const booksCollections = useSelector(selectBooksCollections);
+
+  useEffect(() => {
+    dispatch(fetchFullData());
+  }, [dispatch, setBooksCollections]);
+  return (
+    <>
+      <h1 className="header">Welcome to The Book App</h1>
+      <div className="grid">
+        <div className="leftColumn">
+          <h6 className="columnHeader">BOOK RECOMMENDATIONS</h6>
+          <SidebarLeft />
+        </div>
+        <div className="middleColumn">
+          <h6 className="columnHeader">RECENT REVIEWS</h6>
+          {booksCollections.length > 0 &&
+            booksCollections.map((item) => {
+              if (!item.review) {
+                return;
+              }
```

Comment:
idk if you know but this is called `early-returns` and its a pretty good practice, I really like it. If you dont have the stuff you need for a function, return. don't chain if and elses forever, 👍